frontend/test/metabase/scenarios/custom-column/cc-data-type.cy.spec.js

```diff
@@ -58,6 +58,14 @@ describe("scenarios > question > custom column > data type", () => {
         name: "Datetime Subtract",
         formula: 'datetimeSubtract([Created At], 1, "month")',
       },
+      {
+        name: "ConvertTimezone 2 args",
+        formula: 'convertTimezone([Created At], "Asia/Ho_Chi_Minh", "UTC")',
+      },
+      {
+        name: "ConvertTimezone 1 arg",
+        formula: 'convertTimezone([Created At], "Asia/Ho_Chi_Minh")',
+      },
```

Comment:
"2 args" -> "3 args"
"1 arg" -> "2 args"

Reply:
Changed in 7dd21288b0